README.md

```diff
@@ -1,2 +1,126 @@
 # docs-template
-Docs CI can be automatically added to any SciCat repo as long as it follows the structure here. 
+
+This repository provides a template for the any documentation on the microservices that make up the SciCat project.
+
+It includes:
+
+   *  Preconfigured GitHub Actions workflow for automated documentation deployment
+   *  Custom CSS styling for consistent branding across the SciCat project.
+   *  MkDocs Material theme setup for professional-looking documentation
+   *  Flexibility to accommodate different documentation structures from embedded README.md files to `docs` directories.
+
+# Adding to an existing repository
+
+## Default
+The default way of using this repository is to use it as an action in the repository of interest. This will apply the configuration from [./.github/actions/mkdocs-pages/](./.github/actions/mkdocs-pages/), which contains defaults from `navigations`, `assets` and `styles`
+
+To use this in your repository copy `./.github/workflows/publish-docs.yml` into a folder into the `.github/workflows/` folder in your repository. [./.github/workflows/publish-docs.yml](./.github/workflows/publish-docs.yml) is an example of the use of the action.
+
+To extend the documentation you can use a `docs` directory or a `README.md` file.
+
+### Use a docs directory
+
+Make a `docs` directory in the root directory of your repository and add markdown files to it. To use add your new files to the docs you will need to copy the file [./.github/actions/mkdocs-pages/mkdocs.yml](./.github/actions/mkdocs-pages/mkdocs.yml) to your repository in a `.github/actions/mkdocs` directory. From here add to the `nav` panel to add your new docs.
```

Comment:
I think this section is wrong, if I understand right there is no need to copy the file, but only do what explained in the example